walkthroughs/howto-ingress-gateway/src/cloudwatch-agent/Dockerfile

```diff
@@ -5,15 +5,5 @@ RUN yum install -y \
     shadow-utils \
```

Comment:
I suppose we could just run the default cw-agent image. But that's not necessary to change right now.